### PR TITLE
[CUBRIDMAN-89] Rewrite Java SP manual - prerequisites, data type mapping, fix examples

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CUBRIDMAN-89

This PR rewrites the Java SP manual. The following will be done.
- Re-organize sections
- Add new content and examples to clarify the specification

**Details**
- Add "Prerequisites" section
  - move "Check the cubrid.conf file" section into "Prerequisites"
  - move "Start Java SP Server" section into the "Prerequisites"
- Revise "Data Type Mapping" section
  - revise data type mapping table
  - add implicit data type conversion table
- Revise examples
  - fix import
  - fix code format
  - compile error

### CODE REVIEW

**Review by charischau — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

File: ko/sql/jsp.rst [hunk 1]
```diff
@@ -21,10 +26,26 @@ Java 저장 함수/프로시저를 사용할 때 얻을 수 있는 이점은 다
...
     *   Java를 제외한 다른 언어에서는 저장 함수/프로시저를 지원하지 않는다. CUBRID에서 저장 함수/프로시저는 오직 Java로만 구현 가능하다.
 
+.. _jsp-prerequisites:
+
+기능 사용을 위한 준비
+==============================================
+
+Java 저장함수/프로시저를 사용하기 위해서 다음의 사항이 준비되어 있어야 한다.
+
+*   **cubrid.conf**에 있는 **java_stored_procedure** 값을 **yes** 로 설정해야한다.
```
Comment (hyunikn) [inline ko/sql/jsp.rst]: The range of boldface is not as intened in built web-page. 
Probably, we have to insert a space between '**' and '에'.

File: ko/sql/jsp.rst [hunk 2]
```diff
@@ -21,10 +26,26 @@ Java 저장 함수/프로시저를 사용할 때 얻을 수 있는 이점은 다
...
+*   Java 저장 프로시저/함수를 사용하려는 데이터베이스에 대해 Java 저장 프로시저 서버 (Java SP 서버) 를 시작해야한다.
+
+cubrid.conf 확인
+----------------
+
+**cubrid.conf** 에 있는 **java_stored_procedure** 의 설정값은 **no** 가 기본이다.     
+Java 저장함수/프로시저를 사용하기 위해서는 이 값을 **yes** 로 변경해야 한다. 이 값과 관련한 자세한 설명은 데이터베이스 서버 설정의 `기타 파라미터 <#pm_pm_db_classify_etc_htm>`_ 를 참조한다.
```
Comment (hyunikn) [inline ko/sql/jsp.rst]: The link does not lead me to the '기타 파라미터' page.

File: ko/sql/jsp.rst [hunk 3]
```diff
@@ -6,6 +6,11 @@
 Java 저장 함수/프로시저
 ***********************
 
+.. _jsp-introduction:
+
+Java 저장함수/프로시저 소개
```
Comment (hyunikn) [inline ko/sql/jsp.rst]: In this and many lines below, what about 저장 함수/프로시저 rather than 저장함수/프로시저?

**Review by hyunikn — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

File: ko/sql/jsp.rst (hunk 1 above)
Comment (beyondykk9) [reply ko/sql/jsp.rst]: or insert "\\"

Comment (hgryoo) [reply ko/sql/jsp.rst]: I've fixed it.

File: ko/sql/jsp.rst (hunk 2 above)
Comment (hgryoo) [reply ko/sql/jsp.rst]: I've fixed links for english and korean

File: ko/sql/jsp.rst (hunk 3 above)
Comment (hgryoo) [reply ko/sql/jsp.rst]: fixed

**Review by youngjinj — APPROVED** [review]

**Pull request merged**